test/common/redis/proxy_filter_test.cc

```diff
@@ -272,7 +284,7 @@ TEST_F(RedisProxyFilterTest, ProtocolError) {
   EXPECT_CALL(filter_callbacks_.connection_, close(Network::ConnectionCloseType::NoFlush));
   EXPECT_EQ(Network::FilterStatus::StopIteration, filter_->onData(fake_data));
 
-  EXPECT_EQ(1UL, store_.counter("redis.foo.downstream_cx_protocol_error").value());
+  EXPECT_EQ(1UL, config_->stats_.downstream_cx_protocol_error_.value());
```

Comment:
@mattklein123 I had to fix this in order for the proxy filter test to pass. I am not sure if the changes I did caused some other regression. The stats prefix seems to have been set properly.

Reply:
nm.. fixed the bug..